src/comment.ts

```diff
@@ -145,32 +146,29 @@ const hasChangesetBeenAdded = async (
 }
 
 export const comment = async () => {
-  const {
```

Comment:
It's OK to deconstruct them all together.

Reply:
Since I'm using discriminated unions, typescript does not allow us to deconstruct them here.
If deconstructing them all together is desired, there are two approaches:
1. add variables to both sides of the union, with one of them containing all `undefined`s, another containing all `string`s; or
2. only make `CI_MERGE_REQUEST_SOURCE_BRANCH_NAME` to `string | undefined`, all the others are just `string` in a single type.

The second way is more or less "lying" to typescript, so I would prefer to stick with the current approach or use the first alternative method.